deploy/utils.sh

```diff
@@ -219,8 +219,8 @@ fill_helm_chart() {
     -e "s#{{ TAG }}# ${TAG:-"latest"}#g" \
     -e "s#{{ LABELS }}# ${LABELS:-"app: test-helm"}#g" \
     -e "s#{{ DEBUG }}# ${DEBUG:-"false"}#g" \
-    -e "s#{{ RETRY_COUNT_LIMIT }}# ${RETRY_COUNT_LIMIT:-"3"}#g" \
-    -e "s#{{ RETRY_INTERVAL_SEC }}# ${RETRY_INTERVAL_SEC:-"30"}#g" \
+    -e "s#{{ RETRY_COUNT_LIMIT }}# ${RETRY_COUNT_LIMIT:-"5"}#g" \
```

Comment:
Why not defining the defaults at `utils.sh` and reuse it wherever you need?

Reply:
It is a bit more complicated than that because the existence of the count limit and retry sec is conditional so for me to manipulate them, they need to exist in the environment. So I needed a way to mock defaults because they aren't accessible to me in the environment. 
The places that set them as  `DEFAULT_RETRY_COUNT_LIMIT` in test 25 for example, do not use this function (fill_helm_chart and instead, `fill_helm_chart_no_override_defaults`). `fill_helm_chart_no_override_defaults` is to check that the defaults are taken. Fill_helm_chart function is used to override those defaults